### PR TITLE
Artman Workflow Java sources staging

### PR DESCRIPTION
**Pull request by andreamlin**
This PR is automatically generated by Artmanflow tool.

Includes regen for texttospeech, visionintelligence-v1p1beta1, vision-v1p2beta1 that didn't get regenerated in https://github.com/googleapis/api-client-staging/pull/627

### CODE REVIEW

**Comment by andreamlin** [comment]
PTAL

File: generated/java/gapic-google-cloud-texttospeech-v1beta1/build.gradle [hunk 1]
```diff
@@ -34,4 +34,3 @@ sourceSets {
 }
 
 compileJava.options.encoding = "UTF-8"
-javadoc.options.encoding = "UTF-8"
```
Comment (garrettjonesgoogle) [inline generated/java/gapic-google-cloud-texttospeech-v1beta1/build.gradle]: We need to retain this line. @vam-google added it manually in https://github.com/googleapis/api-client-staging/pull/621 and is adding it to toolkit in https://github.com/googleapis/toolkit/pull/1894/files#diff-6b8cabd49d503be9cdd76bb9d5512b2a which isn't submitted yet.

Comment (vam-google) [reply generated/java/gapic-google-cloud-texttospeech-v1beta1/build.gradle]: The toolkit PR is merged

**Comment by andreamlin** [comment]
@vam-google  PTAL

**Pull request merged**